src/main/java/Garfield.java

```diff
@@ -0,0 +1,235 @@
+import java.util.Scanner;
+import java.io.PrintWriter;
+import java.io.FileWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Garfield {
+
+    public static void Respond(String sentence) {
+        System.out.println("------------------");
+        System.out.print("\t");
+        System.out.println(sentence);
+        System.out.println("------------------");
+    }
+
+    public static void Logo() {
+        String logo = "  ____   ____  ____   _____  ____    ___  _      ___   \n" +
+                " /    | /    ||    \\ |     ||    |  /  _]| |    |   \\  \n" +
+                "|   __||  o  ||  D  )|   __| |  |  /  [_ | |    |    \\ \n" +
+                "|  |  ||     ||    / |  |_   |  | |    _]| |___ |  D  |\n" +
+                "|  |_ ||  _  ||    \\ |   _]  |  | |   [_ |     ||     |\n" +
+                "|     ||  |  ||  .  \\|  |    |  | |     ||     ||     |\n" +
+                "|___,_||__|__||__|\\_||__|   |____||_____||_____||_____|\n" +
+                "\n";
+        System.out.println("---");
+        System.out.println("Summoning\n" + logo);
+    }
+
+    public static void Greet() {
+        Respond("Oh. It's you. Hello. What do you want?");
+    }
+
+    public static void Exit() {
+        Respond("Fine, I'm leaving to find more lasagna.");
+    }
+
+public static List ReadFromFile(String filePath) throws TaskException {
```

Comment:
Avoid lengthy methods. try to keep each method length to about 30 lines of code.